src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -902,16 +901,21 @@ export default class Aircraft {
             expedite: expedite
         });
 
-        let isExpeditingString = '';
-        if (expedite) {
-            isExpeditingString = 'and expedite';
-        }
+        const newAltitude = this.fms.altitudeForCurrentWaypoint();
+        const instruction = radio_trend('altitude', this.altitude, newAltitude);
+        const readback_text = `${instruction} ${newAltitude}`;
+        const readback_verbal = `${instruction} ${radio_altitude(newAltitude)}`;
 
         const readback = {
-            log: `${radioTrendAltitude} ${this.fms.altitudeForCurrentWaypoint()} ${isExpeditingString}`,
-            say: `${radioTrendAltitude} ${currentWaypointRadioAltitude} ${isExpeditingString}`
+            log: `${readback_text}`,
+            say: `${readback_verbal}`
```

Comment:
because the original is a template string, all thats needed here is just the var.
```javascript
// instead of
log: `${readback_text}`,
say: `${readback_verbal}`

// all you need is
log: readback_text,
say: readback_verbal
```